osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -446,6 +453,8 @@ public void UndeleteAll()
         public Task<Live<BeatmapSetInfo>?> ImportAsUpdate(ProgressNotification notification, ImportTask importTask, BeatmapSetInfo original) =>
             beatmapImporter.ImportAsUpdate(notification, importTask, original);
 
+        public void Export(BeatmapSetInfo beatmap) => Task.Run(() => beatmapExporter.ExportAsync(beatmap));
```

Comment:
The threading here is making zero sense. The `ExportAsync()` call (which is *async*) is being wrapped in a `Task.Run()`? And then there is another `Task.Run()` *inside* of `ExportAsync()`? So two TPL threads are now involved? What is going on here?

This should most likely not use `Task.Run()`, and not return `void`. Instead it should return `Task`, and usage sites that don't support async well should use the `.FireAndForget()` extension to get at least a semblance of proper error checking.

Reply:
I'm dizzy that time, sorry